rclcpp/resource/logging.hpp.em

```diff
@@ -108,12 +108,13 @@ def is_supported_feature_combination(feature_combination):
       "First argument to logging macros must be an rclcpp::Logger"); \
 @[ if 'throttle' in feature_combination]@ \
     auto get_time_point = [&clock](rcutils_time_point_value_t * time_point) -> rcutils_ret_t { \
-    try { \
-      *time_point = clock.now().nanoseconds(); \
-    } catch (...) { \
-      return RCUTILS_RET_ERROR; \
-    } \
-      return RCUTILS_RET_OK; \
+      try { \
+        *time_point = clock.now().nanoseconds(); \
+      } catch (...) { \
+        RCUTILS_SAFE_FWRITE_TO_STDERR("could not get current time stamp\n"); \
```

Comment:
Please put some context here, imagine you saw this on the command line from your program, you'd have no idea where to investigate.

Either prefix it with the function that is having the issue or include the file name and line number, e.g.:

https://github.com/ros2/rcutils/blob/3daf12617c2bfe7d6eb49c68e2710d7727ef2e6d/src/error_handling.c#L158-L160

Reply:
@wjwwood unlike the function you linked, these are macros and both `__FILE__` and `__LINE__` will be resolved in the sources in which these appear. I'm fine with that but it's not the same as referring to this file. Which one did you meant?

Reply:
I think you should refer to this file, in the code I linked I did something like `[<pkg>:<file_without_dir.c>]: ....` which is fine. Just some indication as to where this print is coming from. I included the line number because I could but if `__LINE__` will not work correctly in this case, maybe include the macro name instead.

Reply:
Can you check if it's okay @wjwwood?